packages/teams-js/src/public/dialog.ts

```diff
@@ -86,14 +96,14 @@ export namespace dialog {
   }
 
   /**
-   * Fucntion to call when an event is received from the Parent
+   * Register a listener that wil be triggerd when an event is received from the parent
    *
-   * @param type - The event to listen to. Currently the only supported type is 'message'.
-   * @param listener - listener - The listener that will be called.
+   * @remarks
+   * This function is only called from inside of a dialog
+   *
+   * @param listener - The listener that will be triggered.
    */
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  export function registerOnMessageFromParent(listener: (message: any) => void): void {
-    ensureInitialized();
```

Comment:
Inside appWindow, we don't ensure initialization in a similar (or just basically the same function), but I cannot see why. Any ideas? Now I am planning to add it back. Open for suggestions :) 

```
  public addEventListener(type: string, listener: (message: any) => void): void {
    if (type === 'message') {
      registerHandler('messageForChild', listener);
    }
  }
```

Reply:
In a team sync, we decided we should add the ensureInitialized here and in appWindow APIs. I have pushed a change and closing this comment here :)